cmd/dep/ensure.go

```diff
@@ -224,7 +224,10 @@ func (cmd *ensureCommand) Run(ctx *dep.Ctx, args []string) error {
 	// check if vendor exists, because if the locks are the same but
 	// vendor does not exist we should write vendor
 	var writeV bool
-	vendorExists, _ := dep.IsDir(filepath.Join(sw.Root, "vendor"))
+	path := filepath.Join(sw.Root, "vendor")
+	vendorIsDir, _ := dep.IsDir(path)
```

Comment:
i realize the old code didn't check this error, but as long as we're changing it, let's do so here.

it should be fine to skip the next error, though - i can't imagine how `IsEmptyDir()` would error if `IsDir()` doesn't.